tests/validation/tests/v3_api/test_hosted_eks_cluster.py

```diff
@@ -99,6 +101,43 @@ def test_create_hosted_eks_cluster_2():
     create_and_validate_eks_cluster(cluster_config)
 
 
+@ekscredential
+def test_import_eks_cluster():
+    ec2_cloud_credential = get_aws_cloud_credential()
+    display_name = create_resources_eks()
+    cluster_name = random_test_name("test-auto-eks")
+
+    eks_config_temp = {
+        "amazonCredentialSecret": ec2_cloud_credential.id,
+        "displayName": display_name,
+        "imported": True,
+        "privateAccess": False,
+        "publicAccess": False,
+        "region": EKS_REGION,
+        "secretsEncryption": False,
+        "type": "eksclusterconfigspec"
+    }
+
+    cluster_config = {
+        "eksConfig": eks_config_temp,
+        "name": cluster_name,
+        "type": "cluster",
+        "dockerRootDir": "/var/lib/docker",
+        "enableNetworkPolicy": False,
+        "enableClusterAlerting": False,
+        "enableClusterMonitoring": False
+    }
+    create_and_validate_eks_cluster(cluster_config,
+                                    imported=True)
+
+
+def create_resources_eks():
+    cluster_name = resource_prefix + "-ekscluster"
+    AmazonWebServices().create_eks_cluster(cluster_name)
+    AmazonWebServices().wait_for_eks_cluster_state(cluster_name, "ACTIVE")
+    return cluster_name
+
+
 @pytest.fixture(scope='module', autouse="True")
 def create_project_client(request):
```

Comment:
Could we also have this cluster deleted ? From Rancher and from the EKS console after the test is done?

Reply:
The cluster already deleted from Rancher. I will look into how we can delete from EKS console.

Reply:
Addressed changes. The cluster will now be deleted from the EKS console.